enhancements/security/openshift-image-policy.md

```diff
@@ -0,0 +1,700 @@
+But that may be a moving target, with future clusters using an `openshift-release` ClusterImagePolicy, or pivoting to an `openshift-cluster-version` ImagePolicy, or multiple `openshift-*` ImagePolicies, etc.
+The installer can be taught to adjust with each transition, and overlap isn't a problem, because `overrides` entries can be populated before the cluster learns about a new release manifest:
+
+1. Installer sets `overrides` for the `openshift` ClusterImagePolicy (which the cluster uses).
+2. Installer sets `overrides` for the `openshift` ClusterImagePolicy (which the cluster uses) and also for the `openshift-cluster-version` ImagePolicy.
+3. Installer sets `overrides` for the `openshift` ClusterImagePolicy and also for the `openshift-cluster-version` ImagePolicy (which the cluster uses).
+4. Installer sets `overrides` for the `openshift-cluster-version` ImagePolicy (which the cluster uses).
+
+But things get sticky on updates:
+
+1. Installer sets `overrides` for the `openshift` ClusterImagePolicy (which the cluster uses).
+2. Cluster updates to a new release, which adds the `openshift-cluster-version` ImagePolicy and removes the `openshift` ClusterImagePolicy.
+3. But nothing in-cluster pivots `overrides` to include the `openshift-cluster-version` ImagePolicy, so the unsigned cluster-version operator can no longer be pulled.
+
+I think this risk is acceptable, because the installer knob is supposed to be for internal testing of short-lived clusters.
+But if it turns out we need something in-cluster to migrate these settings, we may want to pivot to [have the machine-config operator manage the policy directly](#have-the-machine-config-operator-manage-the-policy-directly) with a more structured in-cluster knob.
+
+#### Transitioning unsigned-release testing
+
+There is a transitional cost to the additional step (`OPENSHIFT_INSTALL_EXPERIMENTAL_DISABLE_IMAGE_POLICY` for installs, `overrides` for updating older clusters) needed to run an unsigned release image for internal testing.
+But that transitional cost is only expected to affect internal use-cases; external users should always be using signed releases.
+
+#### Restricted-network use
+
+Disconnected/restricted-network use is out of scope for this enhancement, so external clusters running `TechPreviewNoUpgrade` and getting this feature may also need the `OPENSHIFT_INSTALL_EXPERIMENTAL_DISABLE_IMAGE_POLICY` or `overrides` workaround.
+It's possible that an ImageDigestMirrorSet like this would be sufficient:
+
+```yaml
+apiVersion: config.openshift.io/v1
+kind: ImageDigestMirrorSet
+metadata:
+  name: openshift-release-payload
+spec:
+  imageDigestMirrors:
+  - mirrors:
+    - example.com/my-mirror/whatever
+    source: quay.io/openshift-release-dev/ocp-release
+```
+
+In any case, it's out of scope for a reason, but until it gets picked up, it might make things more difficult for `TechPreviewNoUpgrade` disconnected/restricted-network clusters.
```

Comment:
Is the main blocker here support in mirror tooling or we need any changes in c/image? @QiWang19 @saschagrunert @mtrmac

Reply:
Also, want to ensure that we capture this as a GA requirement.

Reply:
IIRC this is entirely mirror tooling.